### ISSUE TITLE
Uno.Sample.ControlLibrary.Wasm - The target "Build" does not exist in the project

### ISSUE THREAD

**Issue by MelbourneDeveloper** [unoplatform/Uno.Samples#6]
When I try to build Uno.Sample.ControlLibrary.Wasm in VS 2017 15.9.12, I get

> Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB4057	The target "Build" does not exist in the project.	XamlControlLibrary	C:\Code\Uno.Samples\UI\ControlLibrary\XamlControlLibrary\XamlControlLibrary.csproj	1	

![image](https://user-images.githubusercontent.com/16697547/59643750-4d137000-91ad-11e9-914d-5dc340c0b65f.png)

**Comment by MelbourneDeveloper** [comment]
On Mac:

> /Users/CFindlay/.nuget/packages/uno.sourcegenerationtasks/1.22.0-dev.32/build/net45/Uno.SourceGenerationTasks.targets(3,3): Error MSB4062: The "Uno.SourceGeneratorTasks.SourceGenerationTask_v5d0923b6b600ebff51ba20c89e08ae2d6ce32e5a" task could not be loaded from the assembly /Uno.SourceGeneratorTasks.v5d0923b6b600ebff51ba20c89e08ae2d6ce32e5a.dll. Invalid Image Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. (MSB4062) (XamlControlLibrary)

![image](https://user-images.githubusercontent.com/16697547/59643784-76340080-91ad-11e9-85fc-f2db5bc86ee0.png)

**Comment by ahmedaniss25** [comment]
i fix this problem here : https://www.youtube.com/watch?v=_Jl0q5U3R1A you can follow my step to fix this problem .

**Comment by hawkerm** [comment]
I find that I have to build the UWP head manually first before the WASM builds, but then with a fresh clone of the current repo, I still get this error trying to run the WASM head:
```
System.InvalidOperationException: The SPA default page middleware could not return the default page '/index.html' because it was not found, and no other middleware handled the request.

   at Microsoft.AspNetCore.SpaServices.SpaDefaultPageMiddleware.<>c__DisplayClass0_0.<Attach>b__1(HttpContext context, Func`1 next)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapWhenMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
```

**Comment by jeromelaban** [comment]
This issue is fixed since Uno 4.0
